From 9491773fab574d205992e26d7cc474a01cee97d7 Mon Sep 17 00:00:00 2001
From: Tony Lin <tony.lin@freescale.com>
Date: Thu, 8 Sep 2011 10:24:55 +0800
Subject: ENGR00156232 [mx6q]change cpuinfo keyword

MX[0-9] is the keyword for multimedia applications,
so change the cpuinfo from i.MX 6Quad to MX6 Quad

Signed-off-by: Tony Lin <tony.lin@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabreauto.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabreauto.c b/arch/arm/mach-mx6/board-mx6q_sabreauto.c
index 8f2d31f..2fc366e 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabreauto.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabreauto.c
@@ -912,7 +912,7 @@ static void __init mx6q_reserve(void)
 /*
  * initialize __mach_desc_MX6Q_SABREAUTO data structure.
  */
-MACHINE_START(MX6Q_SABREAUTO, "Freescale i.MX 6Quad SABRE Auto Board")
+MACHINE_START(MX6Q_SABREAUTO, "Freescale MX6 Quad SABRE Auto Board")
 	/* Maintainer: Freescale Semiconductor, Inc. */
 	.boot_params = MX6_PHYS_OFFSET + 0x100,
 	.fixup = fixup_mxc_board,
-- 
1.7.9.5

